docs/fields.md

```diff
@@ -367,6 +367,40 @@ class MyModel(edgy.Model):
 
 ```
 
+Hint you can change the base for the reverse end with embed_parent:
+
+```python
+import edgy
```

Comment:
Put this in a file inside a specific folder inside the `docs_src/` and import it here, please. There are examples in the docs how we do it:

```python
{!> ../docs_src/folder/file_name.py !}
```